salt/modules/nilrt_ip.py

```diff
@@ -216,12 +216,15 @@ def _get_service_info(service):
         data['ipv4']['dns'] = nameservers
     else:
         data['up'] = False
+        data['ipv4'] = {
+            'requestmode': 'disabled'
+        }
 
-    if 'ipv4' in data:
-        data['ipv4']['supportedrequestmodes'] = [
-            'static',
-            'dhcp_linklocal'
-        ]
+    data['ipv4']['supportedrequestmodes'] = [
+        'static',
+        'dhcp_linklocal',
+        'disabled'
```

Comment:
Was switching the order of these intentional?

Reply:
Nope. I can sort them, if it's better